tools/mo/openvino/tools/mo/middle/GatherNDDecomposition.py

```diff
@@ -1,98 +0,0 @@
-# Copyright (C) 2018-2022 Intel Corporation
-# SPDX-License-Identifier: Apache-2.0
-
-import logging as log
-
-import numpy as np
-
-from openvino.tools.mo.ops.gather import Gather
-from openvino.tools.mo.front.common.partial_infer.utils import int64_array
-from openvino.tools.mo.front.tf.graph_utils import create_op_node_with_second_input, create_op_with_const_inputs
-from openvino.tools.mo.graph.graph import Graph, rename_node
-from openvino.tools.mo.middle.replacement import MiddleReplacementPattern
-from openvino.tools.mo.ops.reshape import Reshape
-
-
-class GatherNDDecomposition(MiddleReplacementPattern):
-    """
-    Hot fix for new speech-to-text model enabling while GatherND is not implemented in IE.
-    We can replace GatherND to Reshape + Gather in case when GatherND indices have just one
-    meaningful dimension.
-    TODO: Investigate whether we must replace GatherND with Reshape + Gather always (due to performance benefits)
-          for this particular case or only if the plugin does not support GatherND.
-          And the best place for the transformation is nGraph so we need to move it.
-    """
-    enabled = True
-    force_clean_up = True
-
-    def run_before(self):
-        from openvino.tools.mo.middle.BlockLSTMtoLSTMSequence import BlockLSTMtoLSTMSequence
-        return [BlockLSTMtoLSTMSequence]
-
-    def run_after(self):
```

Comment:
Please do not remove this transformation from MO this because it affects models with BlockLSTM operation like speech_to_text model. I clarified this very recently.
Sorry if I asked you to do this before.

Reply:
No worries, I restored it in MO. :)